wasatch/FeatureIdentificationDevice.py

```diff
@@ -688,85 +885,86 @@ def get_detector_gain_odd(self, update_session_eeprom=False):
     # is 31.1 - 72.0 (I think).
     #
     # @see https://wasatchphotonics.com/api/Wasatch.NET/class_wasatch_n_e_t_1_1_funky_float.html
-    def set_detector_gain(self, gain):
+    def set_detector_gain(self, gain: float) -> SpectrometerResponse:
         raw = self.settings.eeprom.float_to_uint16(gain)
 
         # MZ: note that we SEND gain MSB-LSB, but we READ gain LSB-MSB?!
         log.debug("Send Detector Gain: 0x%04x (%s)", raw, gain)
         self.settings.eeprom.detector_gain = gain
-        return self.send_code(0xb7, raw, label="SET_DETECTOR_GAIN")
+        return self._send_code(0xb7, raw, label="SET_DETECTOR_GAIN")
 
-    def set_detector_gain_odd(self, gain):
+    def set_detector_gain_odd(self, gain: float) -> SpectrometerResponse:
         if not self.settings.is_ingaas():
             log.debug("SET_DETECTOR_GAIN_ODD only supported on InGaAs")
-            return False
+            return SpectrometerResponse(error_lvl=ErrorLevel.low, error_msg="SET_DETECTOR_GAIN_ODD only supported on InGaAs")
 
         raw = self.settings.eeprom.float_to_uint16(gain)
 
         # MZ: note that we SEND gain MSB-LSB, but we READ gain LSB-MSB?!
         log.debug("Send Detector Gain Odd: 0x%04x (%s)", raw, gain)
         self.settings.eeprom.detector_gain_odd = gain
-        return self.send_code(0x9d, raw, label="SET_DETECTOR_GAIN_ODD")
+        return self._send_code(0x9d, raw, label="SET_DETECTOR_GAIN_ODD")
```

Comment:
We can probably change all the {get,set}_detector_{gain,offset}_odd functions to no-ops.  Leave the functions in place for existing code that uses them, but we're probably deprecating the opcodes in FW as well.  We should probably stop actually sending them not only in Wasatch.PY, but probably in Wasatch.NET and Wasatch.VCPP as well. @trestoh